ddtrace/internal/telemetry/data/host.py

```diff
@@ -0,0 +1,21 @@
+import platform
+
+from ...hostname import get_hostname
+from ...runtime.container import get_container_info
+
+
+class Host:
+    def __init__(self):
+        self.os = platform.system()  # type: str
+        self.hostname = get_hostname()  # type: str
+        self.os_version = platform.version()  # type: str
```

Comment:
I am not sure this captures exactly what we want.

There is also `platform.mac_ver()` which helps us get the version of macOS being used, `platform.win32_ver()`.

There is also `platform.platform()` which has more data than we need it in, but should give us a lot ?